### PR TITLE
Use assisted-installer namespace

### PR DESCRIPTION
**Pull request by filanov**
WILL NOT BE PUSHED RIGHT NOW - Will wait until ui is ready. 
And there is pull request in review in bm-inventory

### CODE REVIEW

File: scripts/port_forwarding_loop.sh [hunk 1]
```diff
@@ -3,9 +3,9 @@
 source scripts/utils.sh
 
 function create_port_forwarding() {
-  kubectl --kubeconfig=${KUBECONFIG} port-forward deployment/$3 $1:$2 --address 0.0.0.0
+  kubectl --kubeconfig=${KUBECONFIG} port-forward deployment/$3 $1:$2 --address 0.0.0.0 -n assisted-installer
```
Comment (tsorya) [inline scripts/port_forwarding_loop.sh:6]: this requires ui deployment fix, it must be deployed to the same namespace

Comment (tsorya) [reply scripts/port_forwarding_loop.sh:6]: i think u can create env NAMESPACE="-n assisted-installer" and use it as ${NAMESPACE}

Comment (filanov) [reply scripts/port_forwarding_loop.sh:6]: https://issues.redhat.com/browse/MGMT-761

File: scripts/external_bm_inventory.sh [hunk 2]
```diff
@@ -16,11 +16,11 @@ sudo firewall-cmd --reload
 echo "Starting make run, will start ${SERVICE_NAME}"
 make run
 
-echo "wait till b"
-kubectl rollout restart deployment/${SERVICE_NAME}
+echo "Rollout ${SERVICE_NAME}"
+kubectl rollout restart deployment/${SERVICE_NAME} -n assisted-installer
```
Comment (tsorya) [inline scripts/external_bm_inventory.sh:20]: make assisted-installer env variable as SERVICE_NAME

Comment (filanov) [reply scripts/external_bm_inventory.sh:20]: what?

**Pull request merged**